options/options.go

```diff
@@ -596,6 +596,20 @@ func (opts *ToolOptions) setOptionsFromURI(cs connstring.ConnString) error {
 					return ConflictingArgsErrorFormat("host", strings.Join(cs.Hosts, ","), opts.Host, "--host")
 				}
 			}
+		} else if len(cs.Hosts) > 0 {
+			hostPort := strings.Split(cs.Hosts[0], ":")
```

Comment:
I wasn't sure what exactly to do here since `cs.Hosts` is of type `[]string` while `opts.Host` is just `string`, but figured this might make sense since we just need one member of the replica set. Is that a correct assumption?

Reply:
`opts.Host` should look like `"replSet/foo,bar,baz`. So I think we have to loop through `cs.Hosts` and construct a comma separated list of the hosts without ports for opts.Host. Also if `cs.ReplicaSet` is set, we should prepend `<replSet>/` to the list. 

It looks like `util.CreateConnectionAddrs()` in mongostat expects a comma separated list of hosts.